README.md

```diff
@@ -1,2 +1,27 @@
 # code-genie-bot
 telegram bot for code genie
+
+
+
+## the structure of the bot 
+
+├── code-genie-bot \
+│  # Package initia lization \
+│  # Main bot entry point \
+│ ├── handlers/ \
+│ │  # Handlers package initialization \
+│ │  # Handle /start command \
```

Comment:
no need to specify each command. just say: handle bot commands, or something similar.
the idea is to help anyone reading the code what kind of files will be found in this dir

Reply:
done :)